bedrock/firefox/templates/firefox/new/base.html

```diff
@@ -4,23 +4,17 @@
 
 {% from "macros.html" import google_play_button with context %}
 
-{% add_lang_files "firefox/new/horizon" %}
+{% add_lang_files "firefox/new/onboarding" %}
```

Comment:
The diff looks very confusing, but firefox/new/horizon is being removed, while firefox/new/onboarding moves to firefox/new

Since both templates in onboarding where referencing an extra lang file (firefox/new/onboarding), this move is transparent to us.

Reply:
This should all be good, you can verify by changing locales for each page on the demo (or locally, if you run sync_all)